examples/module1/lesson1/validate-it/validation.ts

```diff
@@ -0,0 +1,26 @@
+import { INTEGER_VALIDATORS, isInteger } from './number-validation';
+
+export function validateNumber(value: string): string {
+  if (!isInteger(value)) {
+    return 'Value is empty or is not an interger.';
```

Comment:
I find it inconsistent - other validators define their own errors messages, but "isInteger" does not.

Reply:
Good point, thanks!